### PR TITLE
Fix: Bump CLI to 6.30

### PR DESCRIPTION
**Pull request by jellypuno**
Signed-off-by: Jessielaine Punongbayan <jessielaine.punongbayan@broadcom.com>

## Proposed changes

Bumping the CLI version to 6.30

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [x] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zFernand0** [comment]
I'm able to create the following profile
```
host: host
port: 1234
rejectUnauthorized: false
protocol: cool
```
However, the CLI prevents me from having a protocol different than `http` or `https`

Do we want to add protocol validation when creating/updating profiles in this PR?

**Comment by jellypuno** [comment]
@zFernand0 I could do that. 👍

**Comment by jellypuno** [comment]
> Do we want to add protocol validation when creating/updating profiles in this PR?

What we are trying to fix is that vulnerability issue. Will it confusing if I add the protocol validation in this PR? Or should I do it in another PR?

**Comment by zFernand0** [comment]
> > Do we want to add protocol validation when creating/updating profiles in this PR?
> 
> What we are trying to fix is that vulnerability issue. Will it confusing if I add the protocol validation in this PR? Or should I do it in another PR?

No strong preference one way or the other. Just something that we may need to handle eventually 😋

**Comment by lauren-li** [comment]
> > Do we want to add protocol validation when creating/updating profiles in this PR?
> 
> What we are trying to fix is that vulnerability issue. Will it confusing if I add the protocol validation in this PR? Or should I do it in another PR?

I think protocol validation would be good to handle in a separate PR, just to keep this PR focused on fixing the vulnerability issue.

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "vscode": "^1.43.0"
   },
   "dependencies": {
-    "@zowe/cli": "^6.23.0",
+    "@zowe/cli": "^6.30.0",
```
Comment (zFernand0) [inline package.json:21]: LGTM!
Don't know if we eventually want to lock it down. i.e. remove the `^`.

**Review by zFernand0 — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
LGTM

**Review by lauren-li — APPROVED** [review]
This looks good to me. All unit and expected integration tests are passing, and the functionalities I tested manually using this PR branch seem to all be working as they did before.

Thanks @jellypuno for this update!

**Pull request merged**